composer.json

```diff
@@ -24,7 +24,7 @@
       }
     ],
     "require": {
-        "wp-coding-standards/wpcs": ">=2.1.0 <2.2.0",
```

Comment:
Yes the range would have installed it if we ran `composer update` which we never do, this locks in `2.1.1`